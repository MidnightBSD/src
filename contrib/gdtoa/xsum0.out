README	c9c34c1	13153
arithchk.c	e37b8a75	4070
dmisc.c	e8d262b6	4712
dtoa.c	f3c132b3	16905
g_Qfmt.c	e5847e9	2870
g__fmt.c	ee5f9be0	2439
g_ddfmt.c	fcf94527	3790
g_dfmt.c	f30e55a9	2533
g_ffmt.c	7c4ea96	2459
g_xLfmt.c	36ee116	2716
g_xfmt.c	c20a5e4	2795
gdtoa.c	364a0d2	17017
gdtoa.h	1eb440de	4810
gdtoaimp.h	6a955ba	19106
gethex.c	1c586a03	5075
gmisc.c	e1a268ea	2114
hd_init.c	cf9a94e	1827
hexnan.c	f53be1da	2988
makefile	e84d078f	2802
misc.c	ea539bdb	14168
smisc.c	f659f036	3685
strtoIQ.c	13ac9f44	1969
strtoId.c	d05a7a6	1961
strtoIdd.c	fa36260d	2135
strtoIf.c	eb75ac99	1905
strtoIg.c	ec59c2fa	3484
strtoIx.c	8f8c9d	1990
strtoIxL.c	1313ff7f	1961
strtod.c	fd6556c8	19985
strtodI.c	e58338e0	4062
strtodg.c	e04b9254	19458
strtof.c	1e7a787a	2202
strtopQ.c	e232c542	2685
strtopd.c	e865dc64	1701
strtopdd.c	1c240126	4540
strtopf.c	1c762782	2196
strtopx.c	17c3fafb	2665
strtopxL.c	ed474cdb	2505
strtorQ.c	126cc92b	2898
strtord.c	1fce44b9	2528
strtordd.c	c32bca0	4979
strtorf.c	fe6a2687	2430
strtorx.c	18389f0c	2888
strtorxL.c	f63fc249	2730
sum.c	dc07b9b	2524
ulp.c	f6db0b4d	1894
