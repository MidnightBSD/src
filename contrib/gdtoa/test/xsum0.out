README	e6ebdc91	2429
Qtest.c	140625e2	4778
dItest.c	8689031	2401
ddtest.c	ea24d330	4943
dtest.c	900d971	4071
dt.c	addb61c	6198
ftest.c	f609ce43	3958
getround.c	f471599	2041
strtoIdSI.c	7bfb88b	49
strtoIddSI.c	72e8852	50
strtodISI.c	ed08b740	49
strtodt.c	17aca428	3213
strtopddSI.c	13e7138d	50
strtorddSI.c	f7e4b1d5	50
xLtest.c	faca328f	4646
xQtest.c	efdea3a2	1549
xtest.c	1f19b87	4858
rtestnos	f94bcdf6	336
testnos	e89999d6	485
testnos1	7e16229	294
testnos3	fa5c8aca	11998
dI.out	d522eef	4369
dIsi.out	1dd6d02f	4350
ddsi.out	1f94bbe2	10251
dd.out	e262456e	40923
dtst.out	ee75f6b9	23308
d.out	f271efc9	28131
f.out	4b0bd51	21207
x.ou0	1cfc5d22	25378
xL.ou0	faa3a741	26363
x.ou1	1a7e9dd4	34587
xL.ou1	418057a	26476
Q.ou0	e4592b85	28742
Q.ou1	ea0b344d	39572
makefile	ebbea1e1	4191
